testsuite/tests/ui/devel/billing/conftest.py

```diff
@@ -36,7 +37,13 @@ def ui_invoice(custom_admin_login, navigator, account, line_items, threescale):
         view.add_item(**line_item)
 
     view.issue()
-    view.charge()
+    try:
+        view.charge()
+    except NoSuchElementException as err:
```

Comment:
Add this to the `InvoiceDetailView`. We would like to use this error handling with all future tests